code/modules/tools/archetypes/tool_extension_variable.dm

```diff
@@ -44,6 +44,6 @@
 	var/tool_name = tool_archetype.name
 	if(tool_archetype.article)
 		tool_name = "\a [tool_name]"
-	to_chat(user, SPAN_NOTICE("You adjust \the [holder] to function as [tool_name]."))
+	to_chat(user, SPAN_NOTICE("You adjust your grip on \the [holder] to use it as [tool_name]."))
```

Comment:
I think the original idea here was something like changing a drill's bit from 'drill' to 'screwdriver' or what have you? Still, this message does work better for non-mechanical tools. I kind of think maybe the variable tool extension just shouldn't be used for those... could make it contextual and pop up a radial menu if it has two? Then again, that could make `IS_TOOL` blocking so you'd have to do it delicately...